src/envelopeCommands.cpp

```diff
@@ -175,11 +177,7 @@ void moveToEnvelopePoint(int direction, bool clearSelection=true) {
 	bool selected;
 	GetEnvelopePointEx(envelope, currentAutomationItem, point, &time, &value, NULL, NULL, &selected);
 	time += offset;
-	if ((direction == 1 && time < now)
-		// If this point is at the cursor, skip it only if it's selected.
-		// This allows you to easily get to a point at the cursor
-		// while still allowing you to move beyond it once you do.
-		|| (direction == 1 && selected && time == now)
+	if ((direction == 1 && time <= now)
```

Comment:
I think this means that if there is a point right at the cursor, we'll always skip it when moving next. This is maybe not what the user expects? We could get around this by skipping only if the point is currentEnvelopePoint.